packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/editingcontext/migration/participants/representation/DiagramAlignmentAttributeMigrationParticipant.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+package org.eclipse.sirius.web.application.editingcontext.migration.participants.representation;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import org.eclipse.sirius.components.collaborative.representations.migration.IRepresentationMigrationParticipant;
+import org.springframework.stereotype.Service;
+
+/**
+ * Remove the attribute alignment from diagram representation and in its content.
+ *
+ * @author gcoutable
+ */
+@Service
+public class DiagramAlignmentAttributeMigrationParticipant implements IRepresentationMigrationParticipant {
+
+    public static final String ALIGNMENT = "alignment";
+
+    @Override
+    public String getVersion() {
+        return "2024.7.7-202407290900";
```

Comment:
- Use the version of the targeted iteration (i.e. 2024.9.0) instead of some random milestone like 2023.7.7
- This date will have to be updated when we will merge this PR